packages/token-lab/src/components/EditableFontFamilyToken/EditableFontFamilyToken.tsx

```diff
@@ -0,0 +1,160 @@
+import { move } from '@dnd-kit/helpers';
+import { DragDropProvider } from '@dnd-kit/react';
+import { useSortable } from '@dnd-kit/react/sortable';
```

Comment:
@drwpow I have this as a draft PR because I'd like to get your thoughts on using this package before going further. Are there alternatives we should explore instead?